app/views/group_offers/_form.html.slim

```diff
@@ -27,6 +27,8 @@
       = f.input :necessary_volunteers, collection: 1..10
     .col-xs-12.col-md-4
       = f.input :volunteer_responsible_state, collection: GroupOffer::VOLUNTEER_RESPONSIBLE_STATES
+    .col-xs-12.col-md-4
```

Comment:
Maybe the category selection fits better either as a separate row below the title or in the same row with the title.